packages/runtime/src/index.ts

```diff
@@ -1,3 +1,4 @@
+import 'reflect-metadata';
```

Comment:
```suggestion
```

Reply:
heh, without that my editor complains that `Reflect.defineMetadata` and `Reflect.getMetadata` do not exists

Reply:
this line should be at the beginning of the entry point of a program, this import should at best be useless.

Reply:
I'm not sure I understand why that should be the case (even if the library in `node_modules`, if I remove the line, the polyfill is not loaded so the build fails), but I'm more than willing to fix it if it's a problem, so where should I put it instead?